drf_braces/utils.py

```diff
@@ -129,7 +144,27 @@ def get_class_name_with_new_suffix(klass, existing_suffix, new_suffix):
     return new_name
 
 
-def get_attr_from_base_classes(bases, attrs, attr, **kwargs):
+def get_attr_from_base_classes(bases, attrs, attr, default=None):
```

Comment:
If this is acceptable, I think we can merge.

The rest of the comments were about docstring